packages/trader/src/AppV2/Components/PayoutInfoModal/payout-info-modal.tsx

```diff
@@ -1,29 +0,0 @@
-import { CaptionText, Modal } from '@deriv-com/quill-ui';
-import { Localize } from '@deriv/translations';
-import React from 'react';
-
-const PayoutInfoModal = ({ body_content }: { body_content: React.ReactNode }) => {
```

Comment:
As I checked, this component wasn't used anywhere

Reply:
yeah I don't remember why it was created but apparently we ain't using it anywhere